### ISSUE TITLE
 Installed this and now I can no longer use Wagtail admin

### ISSUE THREAD

**Issue by serl** [boxed/django-fastdev#29]
Seems like the devs at Wagtail are not using fastdev!

<img width="598" alt="image" src="https://github.com/boxed/django-fastdev/assets/2742655/63d182a0-23dc-45ad-b023-d9c6de48b498">
--
<img width="1638" alt="image" src="https://github.com/boxed/django-fastdev/assets/2742655/e25377c2-d997-4836-b9db-1f0a05176988">

While I'll check with them to fix the undefined tag, I'm sure that's not the only exception.
It would be great if we could exclude specific apps (or directly all external code), with a configuration or something.

TLDR: I think that fastdev should not run outside of the project's boundaries.

Longer explanation: while I totally agree on the fact that undefined variables must explode as they always do everywhere else, I feel that the Django templating language is _meant_ to be more permissive. For me, that simply means that we could find ourselves adding apps to our projects which are written by people not sharing our same views on undefined variables, and are not willing to "fix" what is not a problem to them.

**Comment by boxed** [comment]
Yea, maybe a simple whitelist that is regexes on template names will do. I'm open to a PR for this.

**Comment by Mte90** [comment]
I have a similar issue with https://github.com/unfoldadmin/django-unfold in my case is `Invalid blocks specified:`.
Also if I put django_fastdev as last app I am getting that error for a block of that template of that package that I don't duplicated in my project.

**Comment by Mte90** [comment]
After investigating a bit in my case is that the blocks doesn't exist as it is a package that has a logic to not print them if found them in various HTML files so the logic right now doens't work.

For fastdev is invalid but in reality is not declared or unused.
